test/utils/login-test-user.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -ex
+
+username=test
+password=$(sudo edgexfoundry.secrets-config proxy adduser --user ${username} --useRootToken | jq -r '.password')
+
+vault_token=$(curl -sS "http://localhost:8200/v1/auth/userpass/login/${username}" -d "{\"password\":\"${password}\"}" | jq -r '.auth.client_token')
```

Comment:
I agree that the error handling is done poorly in this script. But I believe adding the if statement to check empty var isn't appropriate because jq returns `null` as output when the item isn't found. Also, we won't be printing the http client error that has resulted from the request so the user can't know why it has failed.

For now, I'll add a flag to make jq return non-zero exit code so the execution of the script stops there. I'll add an issue for more improvements.